tests/components/prosegur/conftest.py

```diff
@@ -0,0 +1,58 @@
+"""Define test fixtures for Prosegur."""
+from unittest.mock import AsyncMock, patch
+
+from pyprosegur.installation import Camera
+import pytest
+
+from homeassistant.components.prosegur import DOMAIN as PROSEGUR_DOMAIN
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+CONTRACT = "1234abcd"
+
+
+@pytest.fixture
+def mock_config_entry() -> MockConfigEntry:
+    """Return the default mocked config entry."""
+    return MockConfigEntry(
+        domain=PROSEGUR_DOMAIN,
+        data={
+            "contract": CONTRACT,
+            CONF_USERNAME: "user@email.com",
+            CONF_PASSWORD: "password",
+            "country": "PT",
+        },
+    )
+
+
+@pytest.fixture
+def mock_install() -> AsyncMock:
+    """Return the mocked alarm install."""
+    install = AsyncMock()
```

Comment:
The installation doesn't need to be an `AsyncMock`. It can be a `MagicMock` or `Mock`.

Reply:
It actually does:

```
    async def async_alarm_arm_home(self, code: str | None = None) -> None:
        """Send arm away command."""
>       await self._installation.arm_partially(self._auth)
E       TypeError: object MagicMock can't be used in 'await' expression

```

Reply:
Either mock the attributes of the installation explicitly or spec the installation according to the original target.

Reply:
done